src/Common/Hooks/Analytics/Analytics.ts

```diff
@@ -0,0 +1,10 @@
+import analytics from '@react-native-firebase/analytics';
+
+export const sendAnalyticsLogEvent = async (
+	event: string,
+	params: {
+		[key: string]: string | number | boolean;
+	} = {},
+) => {
+	await analytics().logEvent(event, params);
```

Comment:
@Takapy1 
下みたいにすれば　()=>void になるから、webとの乖離も解決してEslint も解決しそう！

```suggestion
	analytics().logEvent(event, params).catch(e=>console.error(e));
```

Reply:
@Ayato-kosaka 
試してみます！

Reply:
[1829b2e](https://github.com/Ayato-kosaka/spelieve/pull/607/commits/1829b2ebe2ce649292dd88134b0284807d9ea928)